django/db/models/fields/tuple_lookups.py

```diff
@@ -129,6 +130,20 @@ def as_sql(self, compiler, connection):
         return super().as_sql(compiler, connection)
 
 
+class TupleComparisonMixin:
+    def as_oracle(self, compiler, connection):
```

Comment:
I think it'd be preferable to use a feature flag instead as I wouldn't be surprised if other backends are in the same boat. Maybe `supports_tuple_comparison_subquery`?